Tasks/AzureResourceGroupDeployment/package.json

```diff
@@ -1,9 +1,10 @@
 
 {
-  "name": "DeployAzureResourceGroup",
+  "name": "AzureResourceGroupDeployment",
   "main":"main.js",
   "dependencies": {
     "vsts-task-lib" : "^0.9.20",
-    "vso-node-api": "6.0.1-preview"
+    "vso-node-api": "6.0.1-preview",
+    "strip-json-comments": "1.0.4"
```

Comment:
Do we know what are the nested dependencies this brings? Can you double check the OSS requirements like if the version you are referring is HIBS scanned and if it is tracked in our OSS tool.